x-pack/test/security_solution_endpoint/apps/endpoint/index.ts

```diff
@@ -29,6 +29,6 @@ export default function (providerContext: FtrProviderContext) {
       await ingestManager.setup();
     });
     loadTestFile(require.resolve('./endpoint_list'));
-    loadTestFile(require.resolve('./policy_details'));
+    // loadTestFile(require.resolve('./policy_details'));
```

Comment:
Why skip the `policy_details` test? (maybe you comment it out while testing locally?)

Reply:
Looks good now 😺